portfolio/src/main/webapp/animation.js

```diff
@@ -0,0 +1,44 @@
+particles = [];
```

Comment:
use `let` here? Is this where the variable is declared?

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let